Core/ConfigValues.h

```diff
@@ -71,3 +71,9 @@ enum class SmallDisplayZoom {
 	AUTO = 2,
 	MANUAL = 3,
 };
+
+enum AutoLoadSaveState {
```

Comment:
This will "pollute" the global namespace so that `OFF` is 0.  If you use `enum class AutoLoadSaveState {` instead, it will require `AutoLoadSaveState::OFF` and `OFF` alone won't mean anything.

But in that case, it will require a cast to `(int)`.

-[Unknown]

Reply:
Ah, that explains your cast.  I'll implement and test now.
For future reference should I be squashing my commits as I go along like this, or leaving them perhaps to show how we got here?

Reply:
Either way - usually I worry about if it would give me useful information 6 months from now.  So a typo fix, definitely not.  Whether something is a struct or a class, or enum vs enum class, probably not... but why it's `slot - 3` or why something uses a mutex - probably so.

But generally I err on more commits than less, since bisecting 2000 commits vs 1000 commits is not a big difference anyway.

-[Unknown]